tgui/packages/tgui/interfaces/VotePanel.tsx

```diff
@@ -0,0 +1,201 @@
+import { BooleanLike } from 'common/react';
+import { Box, Icon, Stack, Button, Section, NoticeBox, LabeledList, Collapsible } from '../components';
+import { Window } from '../layouts';
+import { useBackend } from '../backend';
+
+type Vote = {
+  name: string;
+  canBeInitiated: BooleanLike;
+  config: number; // -1 = no config, 0 = config + disabled, 1 = config + enabled
```

Comment:
Make this an enum type instead, like:

```ts
enum VoteConfig {
    None = -1,
    Disabled = 0,
    Enabled = 1,
}
```

Reply:
You should then change every reference of 0, 1, etc to `VoteConfig.Enabled`, etc.

Reply:
ENUMSSSSS YESSS